powershell-adapter/Tests/TestClassResource/0.0.1/TestClassResource.psd1

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'TestClassResource.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
+
+# ID used to uniquely identify this module
+GUID = 'b267fa32-e77d-48e6-9248-676cc6f2327f'
+
+# Author of this module
+Author = 'Microsoft'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = '(c) Microsoft. All rights reserved.'
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = '*'
```

Comment:
Are there any cmdlets being exported?  If so, you should list them explicitly instead of using the wildcard

Reply:
https://github.com/PowerShell/PSDesiredStateConfiguration/issues/101